src/proto/FunctionRpc.proto

```diff
@@ -220,7 +223,12 @@ message CloseSharedMemoryResourcesResponse {
   map<string, bool> close_map_results = 1;
 }
 
-// Host tells the worker to load a Function
+// Host tells the worker to load a list of Functions
+message FunctionLoadRequests {
+  repeated FunctionLoadRequest FunctionLoadRequest = 1;
```

Comment:
Same comment as the one from the host repo PR. Should the type name sound singular and child property be plural (since it is a collection type).

Reply:
Ohhh I just read the comment and understand now, lets keep this plural as it does contain a collection - by bad!

Reply:
Let us discuss this naming over the corresponding host PR and I will update this file once we finalize the naming.